### ISSUE TITLE
Can ExpDecay of learning rate start at some intermediate step?

### ISSUE THREAD

**Issue by cossio** [FluxML/Flux.jl#1815]
I would like to start training a model with a constant learning rate (LR) first, and then after some epochs, start decaying exponentially the LR. I think this is not an uncommon practice.

It would be nice to add this option to `ExpDecay`.

**Comment by ToucheSir** [comment]
This is exactly the kind of thing a library like ParameterSchedulers.jl was designed to handle. See some of the surrounding discussion at https://github.com/FluxML/Flux.jl/search?q=scheduler&type=issues.

**Comment by CarloLucibello** [comment]
Using ParameterSchedulers.jl  this behavior can be obtained with a  `Sequence` scheduler https://darsnack.github.io/ParameterSchedulers.jl/dev/docs/tutorials/complex-schedules.html

That said, a keyword argument is more convenient for these kinds of common cases. Since the PR is already there and it is very simple I'm ok with having it.
